R/downloadFile.R

```diff
@@ -212,7 +215,7 @@ downloadFile <- function(id,
                 filename = shiny::reactive({paste(rootname(), "csv", sep = ".")}),
                 content  = function(file) {
                     if (!is.null(datafxns)) {
-                        writeFile("csv", datafxns$csv(), file, logger,
+                        writeFile("csv", datafxns$csv(), file, row_names, logger,
```

Comment:
- Why not updating these function calls to use paramter names as we use to do usually instead of memorizing each param location?